android/app/build.gradle

```diff
@@ -185,7 +185,7 @@ dependencies {
     implementation 'com.google.firebase:firebase-core:16.0.1'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
-    implementation 'network.mysterium:mobile-node:0.5.3'
+    implementation 'network.mysterium:mobile-node:0.0.0-SNAPSHOT'
```

Comment:
Could you give some background what's happening here? Why aren't we using new release?

Reply:
this is newest release